src/main/java/com/commercetools/sync/commons/utils/SphereClientConfigOptions.java

```diff
@@ -0,0 +1,107 @@
+package com.commercetools.sync.commons.utils;
+
+import io.sphere.sdk.client.SphereClientConfig;
+
+import java.util.concurrent.TimeUnit;
+
+public class SphereClientConfigOptions {
```

Comment:
As we discussed before, `RetryableSphereClientWithExponentialBackoff` would be a better builder instead of passing an additional builder instance, that way you might have a cleaner usage. 
```` java
SphereClientConfig config = SphereClientConfig.of(projectKey, clientId, clientSecret);

SphereClient client = RetryableSphereClientWithExponentialBackoff.of(config)
         .withMaxRetries(10)
         .build();

client.execute(command...);

````